### DISCUSSION TITLE
What is the merge with upstream schedule?

### DISCUSSION THREAD

**Discussion by ujl123** [machineminded/Fooocus-inswapper#29] [General]
Forgive me (and correct me) as I don't really understand the git terminology, but how often do you merge with the upstream project? I ask because I'm following a ticket regarding using loras in prompts that's being worked in the upstream project, and was wondering if it will miss your next merge cycle or not.

Thanks!

**Comment by ujl123** [comment]
@machineminded  just to flag you here in case discussions don't generate a notification. Thanks!

**Comment by machineminded** [comment]
Sorry, didn't see this one.   Basically I keep an eye on the main repo and merge with main ASAP.  I just merged with main in the last day or so and pushed it up.

**Comment by machineminded** [comment]
As for prompt based LoRAs, I haven't seen that yet but can check for you later this evening.

**Comment by machineminded** [comment]
@ujl123  Would you be able to share the issue number for the prompt-based LoRA?  I can keep an eye on it.

**Reply by ujl123** [reply]
@machineminded  It's this one specifically : https://github.com/lllyasviel/Fooocus/pull/2323

**Reply by machineminded** [reply]
Looks like it is almost ready to be merged.  I merge whenever `main` on Illyasviel's repo is updated, so hopefully we'll get this soon :)
